php/libraries/Sex.class.inc

```diff
@@ -0,0 +1,71 @@
+<?php declare(strict_types=1);
```

Comment:
I'm a little less convinced that this one belongs in the Candidate namespace, but it probably does.